tests/utils/test_password_manager.py

```diff
@@ -231,3 +232,49 @@ def test_get_http_auth_from_environment_variables(
 
     assert auth["username"] == "bar"
     assert auth["password"] == "baz"
+
+
+def test_get_pypi_token_with_env_var_positive(
+    config: Config, with_simple_keyring: None, dummy_keyring: DummyBackend
+):
+    sample_token = "sampletoken-1234"
+    repo_name = "foo"
+    manager = PasswordManager(config)
+
+    with mock.patch.dict(
+        os.environ,
+        {f"POETRY-PYPI-TOKEN-{repo_name}".upper().replace("-", "_"): sample_token},
+    ):
+        result_token = manager.get_pypi_token(repo_name)
+
+    assert result_token == sample_token
+
+
+def test_get_pypi_token_with_env_var_negative(
```

Comment:
this test seems worthless - it doesn't prove anything not already proved by the previous test

Reply:
deleted, please check @dimbleby